portfolio/src/main/webapp/index.html

```diff
@@ -2,14 +2,21 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Daekoon's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
```

Comment:
@daekoon, I have informational comment to improve performance better. See if you can read and understand the article below.
https://flaviocopes.com/javascript-async-defer/

Reply:
Thank you for the read! I have added a defer tag to the script loading.